aws/logs_monitoring/steps/handlers/s3_handler.py

```diff
@@ -296,6 +296,7 @@ def _extract_other_logs(self):
                     "DD_MULTILINE_LOG_REGEX_PATTERN %s did not match start of file, splitting by line",
```

Comment:
Even though the log says "splitting by line", the `self.data_store.data` is not split by line for now, actually.

Reply:
Nice catch, beware that String `splitlines` and Bytes don't behave exactly the same (cf [here](https://github.com/DataDog/datadog-serverless-functions/blob/6518604/aws/logs_monitoring/steps/handlers/s3_handler.py#L307)). I guess for an error scenario it's good enough.